packages/protocol/contracts/governance/Validators.sol

```diff
@@ -596,12 +516,12 @@ contract Validators is
     require(commission <= FixidityLib.fixed1().unwrap(), "Commission can't be greater than 100%");
     address account = getLockedGold().getAccountFromActiveValidator(msg.sender);
     require(!isValidator(account) && !isValidatorGroup(account));
-    require(meetsValidatorGroupBalanceRequirements(account));
-
+    uint256 lockedGoldBalance = getLockedGold().getAccountTotalLockedGold(account);
```

Comment:
No, only because the group hasn't been registered yet, so the requirement is `0`. If we did this check after registration (i.e. moved this line to the end of the function) then we could use `meetsAccountLockedGoldRequirements`